### PR TITLE
Add card-back viewing functionality

### PR DESCRIPTION
**Pull request by yong-jie**
Currently looks like shit, but will do for now. Naturally, tests that rely on the presence of the old browserpanel will fail, so I'll push the fix in a separate PR

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -60,7 +60,7 @@
             <StackPane fx:id="cardListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
           </VBox>
 
-          <StackPane fx:id="browserPlaceholder" prefWidth="340" >
+          <StackPane fx:id="cardBackPlaceholder" prefWidth="340" >
```
Comment (jethrokuan) [inline src/main/resources/view/MainWindow.fxml]: no need to call it `Placeholder` anymore

**Comment by jethrokuan** [comment]
Feel free to delete all browser panel related tests.

File: src/main/java/seedu/address/ui/CardBack.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        super(FXML);
+        registerAsAnEventHandler(this);
+    }
+
+    @Subscribe
+    private void handleCardListPanelSelectionChangedEvent(CardListPanelSelectionChangedEvent event) {
+        logger.info(LogsCenter.getEventHandlingLogMessage(event));
+        cardBack.setText(event.getNewSelection().card.getBack());
+    }
+}
```
Comment (jethrokuan) [inline src/main/java/seedu/address/ui/CardBack.java]: you didn't close the author tag

Comment (yong-jie) [reply src/main/java/seedu/address/ui/CardBack.java]: All is good i think. The website mentions that 
```
The code up to the next //@@author tag or the end of the file (whichever comes first) will be considered as was written by that author.
```

Comment (jethrokuan) [reply src/main/java/seedu/address/ui/CardBack.java]: :+1:

**Pull request merged**